Agile Development/Burndown Chart/script.py

```diff
@@ -0,0 +1,132 @@
+import pprint
+import requests
+import jpholiday
+import datetime
+import matplotlib.dates as mdates
+import pandas as pd
+import matplotlib.pyplot as plt
+import urllib.request
+import urllib.parse
+import json
+import datetime
+from pytz import timezone
```

Comment:
Can you add a requirements.txt (as well as info on how to use it in README.md)? requests, jpholiday, matplotlib, pandas, and pytz aren't built-in packages and need to be installed with pip.